ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10.yaml

```diff
@@ -88,6 +88,7 @@ tests:
   steps:
     cluster_profile: gcp
     env:
+      CREATE_SSH_BASTION: "no"
```

Comment:
Maybe we can drop `CREATE_SSH_BASTION` insert in this file, and in script, when `CREATE_SSH_BASTION` is not defined, we can treat that as "no" by default, so that we do not need to introduce this new var anywhere.

Reply:
but when will it be defined then?

Reply:
When the workflow indeed need a bastion, we can define and set it to "yes".

Reply:
If no any step defines corresponding param, trying to set it a value would get error like 'parameter \"FOO_XYZ\" is overridden in [workflow/cucushift-installer-rehearse-gcp-upi-private] but not declared in any step', so your solution isn't acceptable. 

FYI the workflow I tried with: 

workflow:
  as: cucushift-installer-rehearse-gcp-upi-private
  steps: 
    pre:
    - chain: upi-gcp-pre
    post:
    - chain: upi-gcp-post
    env:
      DISCONNECTED_NETWORK: "no"
      PUBLISH_STRATEGY: "Internal"
      FOO_XYZ: 2021
  documentation: |-
    This is the workflow to trigger Prow's rehearsal test when submitting installer steps/chain/workflow